### PR TITLE
Fix some bugs with corpses

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

**Review by DolceTriade — APPROVED** [review]

File: src/shared/bg_utilities.cpp [hunk 1]
```diff
@@ -165,3 +165,9 @@ void RemoveFlag(int &flags, int flag) {
 void ToggleFlag(int &flags, int flag) {
 	flags ^= flag;
 }
+
+// sets the first argument the new anim, but toogling the toogle bit
+void BG_NewAnim(int &anim, int newAnim) {
```
Comment (slipher) [inline src/shared/bg_utilities.cpp:170]: It is not good style to use a reference here - just take one input and return something.

But anyway I don't think this commit is all that helpful in general. It's hard to tell what this function does looking at the name; I'd rather just have the code right there at the callsite, as it is now.

**Comment by necessarily-equal** [comment]
I've removed the commit for now

**Comment by slipher** [comment]
It seems, you removed the wrong commit.

**Comment by necessarily-equal** [comment]
fixed

**Pull request merged**